uber/tests/models/test_job.py

```diff
@@ -46,7 +46,8 @@ def test_non_volunteer(self, session):
 
     def test_restricted(self, session):
         assert session.assign(session.staff_two.id, session.job_six.id)
-        session.staff_two.trusted = True
+        session.staff_two.trusted_depts = str(c.CONSOLE)
+        # TODO: update
         session.commit()
         assert not session.assign(session.staff_two.id, session.job_six.id)
 
```

Comment:
ok gotcha, that makes sense. would you say this is the right call then:
Add staffing_adjustment to 
1) if someone is marked as trusted but they're not in that dept, mark them as being in that dept
2) if someone is removed from an assigned department but still marked as trusted, we remove their trusted status as well.